lblod/frontend-gelinkt-notuleren
# GN-4599: prevent confirmation dialog from popping up twice.

### Overview
This PR solves an issue with the confirmation dialog popping up twice upon leaving an edit route.
Specifically, it updates the `isLoadingRoute` to more correctly check if a `RouteInfo` object represents a `loading` route.
The `isLoadingRoute` function now checks for the following possible syntaxes:
- `loading`
- `${routename}-loading`
- `${routename}_loading` (new)

##### connected issues and PRs:
[GN-4599](https://binnenland.atlassian.net/browse/GN-4599?atlOrigin=eyJpIjoiMDUxNjYyYjU0OTgyNDM1OTk1ZGZmZDJjZjNhM2IxMTMiLCJwIjoiaiJ9)

### How to test/reproduce
- Open an agendapoint/regulatory statement
- Type some text without saving
- Leave the route
- Ensure that the confirmation dialog only pops up once.

### Challenges/uncertainties
The `isLoadingRoute` function is quite volatile. If something changes to the internal naming of ember loading routes, the function would need to be updated. 
Another possibility is to remove the `routeWillChange` listener after the user has pressed `ok` in the confirmation dialog.

### Checks PR readiness
